twilio/endpoint.bal

```diff
@@ -141,21 +141,27 @@ public client class Client {
     #
     # + fromNo - Mobile number which the voice call should be send from
     # + toNo - Mobile number which the voice call should be received to
-    # + twiml - TwiML URL which the response of the voice call is stated
+    # + voiceCallInput - What should be heard when the other party picks up the phone (a Url that returns TwiML Voice instructions or 
+    #            inline message. example: "http://demo.twilio.com/docs/voice.xml" or "Thank you for calling")
     # + statusCallback - (optional) StatusCallback record which contains the callback url and the events whose status 
     #                     needs to be delivered.
     # + return - If success, returns voice call response object with basic details, else returns error
     @display {label: "Make a voice call"}
     remote isolated function makeVoiceCall(@display {label: "Caller Number"} string fromNo, 
                                            @display {label: "Callee Number"}string toNo, 
-                                           @display {label: "TwiML URL"} string twiml, 
+                                           @display {label: "Voice Message and Type"} VoiceCallInput voiceCallInput,  
```

Comment:
won't `Input` be enough? We know operation is `Make a voice call` and the generated form will have `type` and `input` fields.

Reply:
Yes, I did the requested change